autoload/fzf/vim.vim

```diff
@@ -76,6 +75,11 @@ endfunction
 
 " [[options to wrap], [preview window expression], [toggle-preview keys...]]
 function! fzf#vim#with_preview(...)
+  let bash_path = exepath('bash')
+  if empty(bash_path)
+    throw 'bash is not in PATH'
+  endif
+  let is_wsl_bash = bash_path =~? 'Windows[/\\]system32[/\\]bash.exe$'
```

Comment:
Can wsl bash appear in another directory?

Reply:
There are other commands you can invoke which will result in WSL running, such as `wsl` or distro-specific exes, but AFAIK they normally won't be named bash.exe, so `system32/bash.exe` is what you'll get from calling `bash`. (note: I work for Microsoft, but not on anything related to WSL, so I'm not an authoritative source on this)

As an example of distro-specific exes, I have [VoidWSL](https://github.com/am11/VoidWSL) installed at `c:/wsl/void.exe` and can run commands there such as `c:/wsl/void.exe run path/to/preview.sh /mnt/c/file/to/preview`.

It might be nice to be able to configure the command I want to use if I want to use a specific distro or if I just want to use git bash without rearranging my PATH.